x-pack/test/functional/apps/transform/cloning.ts

```diff
@@ -126,27 +170,41 @@ export default function ({ getService }: FtrProviderContext) {
           await transform.wizard.assertQueryInputExists();
           await transform.wizard.assertQueryValue('');
 
-          await transform.testExecution.logTestStep(
-            'should show the pre-filled group-by configuration'
-          );
-          await transform.wizard.assertGroupByEntryExists(
-            testData.expected.groupBy.index,
-            testData.expected.groupBy.label
-          );
-
-          await transform.testExecution.logTestStep(
-            'should show the pre-filled aggs configuration'
-          );
-          await transform.wizard.assertAggregationEntryExists(
-            testData.expected.aggs.index,
-            testData.expected.aggs.label
-          );
+          // assert define step form
+          if (isPivotTransform(testData.originalConfig)) {
+            await transform.testExecution.logTestStep(
+              'should show the pre-filled group-by configuration'
+            );
+            await transform.wizard.assertGroupByEntryExists(
+              // @ts-ignore
```

Comment:
Nit: Any chance we could get rid of the ignores?

Reply:
It requires some changes in the test data types. Hope we'll come with a solution with Robert